CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json

```
@@ -0,0 +1,17 @@
+{
+	    "version": "v1.0",
+	    "author": "Baidu USA GAIT LEOPARD team: Baopu Li, Zhiyu Cheng, Jiazhuo Wang, Haofeng Kou, Yingze Bao", 
+	    "authorEmail": "baopuli@baidu.com, zhiyucheng@baidu.com, jiazhuowang@baidu.com, haofengkou@baidu.com, yingzebao@baidu.com",
+	    "framework": "PyTorch torch-1.0.1.post2-cp35-cp35m-linux_x86_64.whl",

```

Thanks for the detailed version information. Could you truncate it to "PyTorch v1.0.1" for the `framework` field and add a `pytorchWheel` field under `misc` with "torch-1.0.1.post2-cp35-cp35m-linux_x86_64.whl"

```suggestion
	    "framework": "PyTorch v1.0.1",
```

---

yes, we truncated it as advised.